src/test/java/uk/gov/companieshouse/controller/AppealControllerTest.java

```diff
@@ -0,0 +1,200 @@
+package uk.gov.companieshouse.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import uk.gov.companieshouse.exception.AppealNotFoundException;
+import uk.gov.companieshouse.model.Appeal;
+import uk.gov.companieshouse.model.OtherReason;
+import uk.gov.companieshouse.model.PenaltyIdentifier;
+import uk.gov.companieshouse.model.Reason;
+import uk.gov.companieshouse.repository.AppealRepository;
+import uk.gov.companieshouse.service.AppealService;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.header;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@RunWith(SpringRunner.class)
+@WebMvcTest
+public class AppealControllerTest {
+
+    private static final String APPEALS_URI = "/companies/{company-id}/appeals";
+
+    private static final String IDENTITY_HEADER = "ERIC-identity";
+
+    private static final String TEST_USER_ID = "1234";
+    private static final String TEST_COMPANY_ID = "12345678";
+    private static final String TEST_RESOURCE_ID = "1";
+    private static final String TEST_PENALTY_REFERENCE = "A12345678";
+    private static final String TEST_REASON_TITLE = "This is a title";
+    private static final String TEST_REASON_DESCRIPTION = "This is a description";
+
+    @MockBean
+    private AppealService appealService;
+
+    @MockBean
+    private AppealRepository appealRepository;
```

Comment:
Seems to be never user. Is that required in the light of mocking `private AppealService appealService`?

Reply:
removed this as it wasn't used